polaris-icons/icons/PurchaseIcon.yml

```diff
@@ -0,0 +1,10 @@
+name: Purchase
+description: Used for most purchase transactions
+keywords:
+  - finances
+  - financial accounts
+  - transactions
+authors:
+  - Eduardo Mujica
+date_added: 2024-06-07
+date_modified: 2024-06-07
```

Comment:
This seems like a duplicate of the receipt icon

Reply:
👋 hey we needed an icon to label purchases, which are an instance of the transaction object in Finances. Transactions in turn contain receipts which are issued by Stripe.